lib/seek/isa_templates/template_attributes_schema.json

```diff
@@ -1,112 +1,140 @@
-		"description": { "type": "string" },
-		"required": {
-			"anyOf": [
-				{
-					"type": "boolean"
-				},
-				{
-					"type": "null"
-				}
-			]
-		},
-		"ontology": {
-			"anyOf": [
-				{
-					"type": "object",
-					"properties": {
-						"name": { "type": "string" },
-						"version": {
-							"anyOf": [{ "type": "string" }, { "type": "number" }]
-						},
-						"description": { "type": "string" },
-						"rootTermURI": { "type": "string", "format": "uri" }
-					}
-				},
-				{
-					"type": "null"
-				}
-			]
-		},
-		"dataType": {
-			"type": "string",
-			"enum": [
-				"Date time",
-				"Date",
-				"Real number",
-				"Integer",
-				"Web link",
-				"Email address",
-				"Text",
-				"String",
-				"Boolean",
-				"Registered Sample",
-				"Registered Sample List",
-				"Controlled Vocabulary",
-				"URI",
-				"Registered Data file",
-				"ENA custom date"
-			]
-		},
-		"CVList": {
-			"anyOf": [
-				{
-					"type": "array"
-				},
-				{
-					"type": "null"
-				}
-			]
-		},
-		"isaTag": {
-			"anyOf": [
-				{
-					"type": "string",
-					"enum": [
-						"source",
-						"sample",
-						"protocol",
-						"source_characteristic",
-						"sample_characteristic",
-						"other_material",
-						"other_material_characteristic",
-						"data_file",
-						"data_file_comment",
-						"parameter_value"
-					]
-				},
-				{
-					"type": "null"
-				}
-			]
-		},
-		"title": {
-			"anyOf": [
-				{
-					"type": "boolean"
-				},
-				{
-					"type": "null"
-				}
-			]
-		}
-	},
-	"additionalProperties": false
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "ISA tempalte attribute schema",
+  "description": "JSON-schema representing an ISA template attribute in DataHub",
+  "type": "object",
+  "properties": {
+    "iri": {
+      "anyOf": [
+        {
+          "type": "string",
+          "format": "uri"
+        },
+        {
+          "type": "null"
+        }
+      ]
+    },
+    "name": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string"
+    },
+    "required": {
+      "anyOf": [
+        {
+          "type": "boolean"
+        },
+        {
+          "type": "null"
+        }
+      ]
+    },
+    "ontology": {
+      "anyOf": [
+        {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string"
+            },
+            "version": {
+              "anyOf": [
+                {
+                  "type": "string"
+                },
+                {
+                  "type": "number"
+                }
+              ]
+            },
+            "description": {
+              "type": "string"
+            },
+            "rootTermURI": {
+              "type": "string",
+              "format": "uri"
+            }
+          }
+        },
+        {
+          "type": "null"
+        }
+      ]
+    },
+    "dataType": {
+      "type": "string",
+      "enum": [
+        "Date time",
```

Comment:
these attribute types aren't necessarily locked down, and can be changed or new ones added through the database - meaning the schema will need change, which would be readonly in Docker. Not a blocker for this PR, but something to concider changing. This is something that needed changing for the extended metadata type schema, and it would be worth trying to align them

Reply:
I agree! It already happened a couple of times...